README.md

```diff
@@ -200,27 +200,33 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
    If this is your first time running InstructLab, it's best to start with `ilab config init` to create the environment.
 
    Options:
-   --config PATH  Path to a configuration file.  [default: config.yaml]
+   --config PATH  Path to a configuration file.  [default:
+                  /home/cdoern/.config/instructlab/config.yaml]
+   -v, --verbose  Enable debug logging (repeat for even more verbosity)
    --version      Show the version and exit.
    --help         Show this message and exit.
 
-   Command:
-      config      Command group for Interacting with the Config of InstructLab
-      data        Command group for Interacting with the Data of generated by...
-      model       Command group for Interacting with the Models in InstructLab
-      sysinfo     Print system information
-      taxonomy    Command group for Interacting with the Taxonomy in InstructLab
+   Commands:
+   config    Command Group for Interacting with the Config of InstructLab.
```

Comment:
The capitalization here is really strange, why are words like group, interacting, config, etc. capitalized, while others like 'command', 'related', 'generated', etc. are not? 

If we are adjusting the capitalization in this PR then we should normalize all of the sentences so that the capitalization is in line with standard English rules.

Reply:
I agree, but what's important here is that it matches what's in the CLI

Reply:
Ack, it sounds like this is something we'll want to handle in a separate PR then post-0.18.0. 

I've created an issue for it here: https://github.com/instructlab/instructlab/issues/2064